routes/main/user/google/index.js

```diff
@@ -0,0 +1,16 @@
+const MainUserGoogleRouter = require("express").Router();
```

Comment:
I don't think there is a need for google directory, I think those routes can live directly in user file